gulpfile.js

```diff
@@ -185,6 +185,19 @@ function createWebpackConfig(defines, output) {
     !bundleDefines.TESTING;
   var skipBabel = bundleDefines.SKIP_BABEL;
 
+  // `core-js` (see https://github.com/zloirock/core-js/issues/514),
+  // `web-streams-polyfill` (already using a transpiled file), and
+  // `src/core/{glyphlist,unicode}.js` (Babel is too slow for those when
+  // source-maps are enabled) should be excluded from processing.
+  const babelExcludes = [
+    "node_modules[\\\\\\/]core-js",
+    "node_modules[\\\\\\/]web-streams-polyfill",
+  ];
+  if (enableSourceMaps) {
+    babelExcludes.push("src[\\\\\\/]core[\\\\\\/](glyphlist|unicode)");
+  }
+  const babelExcludeRegExp = new RegExp("(" + babelExcludes.join("|") + ")");
```

Comment:
I think we can use a template string here for slightly improved readability.

Reply:
Sure, why not.